pkg/daemon/command/command.go

```diff
@@ -122,12 +124,23 @@ func WithThreadNum(num int) Opt {
 	}
 }
 
-func WithConfig(config string) Opt {
+func WithConfig(id string) Opt {
 	return func(cmd *DaemonCommand) {
+		config := strings.Replace("/api/v1/daemons/{id}/config", "{id}", id, 1)
```

Comment:
Should we use `strings.Replace()` here?
`fmt.Sprintf("/api/v1/daemons/%s/config", id)` also works.

Reply:
Okay, I'll take this better approach